docs/ruleset_functions/get_building_segment_SWH_bat.md

```diff
@@ -0,0 +1,64 @@
+## get_building_segment_SWH_bat
+
+Description: This function determines the SWH BAT for the given building segment.
+
+Inputs:
+- **RMD**
+- **building_segment**
+
+Returns:
+- **building_segment_swh_bat**: one of the ServiceWaterHeatingSpaceOptions2019ASHRAE901 options
+
+Function Call:
+
+- get_energy_required_to_heat_swh_use  
+- get_SWH_uses_associated_with_each_building_segment    
+- get_component_by_ID  
+
+Data Lookup: None
+
+Logic:
+
+- set the building_segment_swh_bat to nil: `building_segment_swh_bat = "UNDETERMINED"`
+- if the building_segment.service_water_heating_building_area_type exists set the swh_bat to the one given: `if building_segment.service_water_heating_building_area_type:`
```

Comment:
I can't find the ```service_water_heating_building_area_type``` under ```building_segment```. Can you clarify this?

Reply:
https://github.com/open229/ruleset-model-description-schema/issues/264